pkg/metadata/annotations.go

```diff
@@ -32,9 +32,10 @@ type OpflexGroup struct {
 
 // annotation type for service endpoint information
 type ServiceEndpoint struct {
-	Mac  string `json:"mac,omitempty"`
-	Ipv4 net.IP `json:"ipv4,omitempty"`
-	Ipv6 net.IP `json:"ipv6,omitempty"`
+	HealthGroupDn   string `json:"health-group,omitempty"`
```

Comment:
I think we should use json:"-" for HealthGroupDn as we don't need to write this as json.

Reply:
apicRedirectDst() relys on this to be an empty string for cases where IPSLA is not enabled. Do you see an issue by leaving it this way?

Reply:
That shouldn't be affected. This matters only when we call json.Marshal directly on this struct.

Reply:
ok, I just took care of it.